.circleci/config.yml

```diff
@@ -105,9 +92,32 @@ jobs:
       # Run Common Tests.
       - run: ./common/run_tests.sh
 
+      - run: ./prepare_image.sh -i smasher -s workers -d localhost:5000
+
+      # Run Smasher tests.
+      - run: sudo chown -R circleci:circleci workers/test_volume/
+      - run:
+          command: .circleci/filter_tests.sh -t smasher
+          # Smashing can apparently take a long time on CI
+          no_output_timeout: 1h
+
+      # Run Janitor tests.
+      - run: sudo chown -R circleci:circleci workers/test_volume/
+      - run:
+          command: .circleci/filter_tests.sh -t janitor
+          no_output_timeout: 1h
+
+      # Run Compendia tests.
+      - run: sudo chown -R circleci:circleci workers/test_volume/
+      - run:
+          command: .circleci/filter_tests.sh -t compendia
+          no_output_timeout: 1h
+
+
   # This tests workers tests tagged as 'salmon'
   salmon_and_api_tests:
     working_directory: ~/refinebio
+    # resource_class: xlarge
```

Comment:
What is this?

Reply:
Trying to get boxes big enough to actually run these tests. Circle told me this should work, but I got an error saying xlarge was an invalid setting. I'm in contact with them, but you're right: I should sort this out first.